src/DI/ContainerLoader.php

```diff
@@ -108,7 +108,7 @@ protected function generate($class, $generator)
 	{
 		$compiler = new Compiler;
 		$compiler->getContainerBuilder()->setClassName($class);
-		$code = call_user_func_array($generator, [& $compiler]);
+		$code = $generator(...[& $compiler]);
```

Comment:
@hrach: That is a syntax error in PHP 7.
What format exactly? I think only static methods? That would be eventually solved when we switch to PHP 7.